rpc/tests/p2p.rs

```diff
@@ -0,0 +1,296 @@
+use crate::utils::client::{new_test_client, AuthLevel};
+use celestia_rpc::p2p;
+use celestia_rpc::prelude::*;
+use tokio::time::{sleep, Duration};
+
+pub mod utils;
+
+async fn rpc_call_delay() {
+    // delay for RPC calls like connect/close to let node finish the operation before we query it
+    // again. Below 150 ms I start getting intermittent failures.
+    sleep(Duration::from_millis(150)).await;
+}
+
+#[tokio::test]
+async fn info_test() {
+    let client = new_test_client(AuthLevel::Admin).await.unwrap();
+    client.p2p_info().await.expect("Failed to get node info");
+}
+
+#[tokio::test]
+async fn add_remove_peer_test() {
+    //! add and then remove a peer, testing outputs from `p2p.Peers` and `p2p.Connectedness`
+    let (addr_info, task) = utils::tiny_node::start_tiny_node()
+        .await
+        .expect("failed to spin up second node");
+    log::debug!("{addr_info:?}");
+    let client = new_test_client(AuthLevel::Admin).await.unwrap();
+
+    let initial_peers = client
+        .p2p_peers()
+        .await
+        .expect("failed to get initial peer list");
+    assert!(initial_peers.contains(&addr_info.id) == false);
```

Comment:
use `assert_eq`. check other asserts too

Reply:
switched `assert!` to `assert_eq!` in tests